<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remote</class>
 <widget class="QWidget" name="Remote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>598</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Remote Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="power">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Q</string>
       </property>
       <property name="text">
        <string>Power</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="one">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>1</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="two">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>2</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="three">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>3</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="four">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>4</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="five">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>5</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="six">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>6</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="seven">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>7</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="eight">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>8</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="nine">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>9</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="zero">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>0</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pip">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="text">
        <string>PIP</string>
       </property>
       <property name="toolTip">
        <string>I</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="enter">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="text">
        <string>ENT</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QPushButton" name="rewind">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>&lt; (previous track)</string>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>P</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fastForward">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>&gt;(next track)</string>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QPushButton" name="rec">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>R</string>
       </property>
       <property name="text">
        <string>Rec</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>S</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>Space</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QPushButton" name="src">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="text">
        <string>Src</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disp">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>D</string>
       </property>
       <property name="text">
        <string>Dsp</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sleep">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="text">
        <string>Sleep</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="guide">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>G</string>
       </property>
       <property name="text">
        <string>Guide</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QPushButton" name="up">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>Up Arrow</string>
       </property>
       <property name="text">
        <string>^</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="left">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>Left Arrow</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="right">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>Right Arrow</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="down">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>Down Arrow</string>
       </property>
       <property name="text">
        <string>v</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="menu">
       <property name="toolTip">
        <string>M</string>
       </property>
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="text">
        <string>Menu</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>O</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="volumeUp">
	  <property name="focusPolicy">
	    <enum>Qt::NoFocus</enum>
	  </property>
         <property name="toolTip">
          <string>]</string>
         </property>
         <property name="text">
          <string>V+</string>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="volumeDown">
	  <property name="focusPolicy">
	    <enum>Qt::NoFocus</enum>
	  </property>
         <property name="toolTip">
          <string>[</string>
         </property>
         <property name="text">
          <string>V-</string>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="brightnessUp">
	  <property name="focusPolicy">
	    <enum>Qt::NoFocus</enum>
	  </property>
         <property name="toolTip">
          <string>}</string>
         </property>
         <property name="text">
          <string>Ch+</string>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="brightnessDown">
	  <property name="focusPolicy">
	    <enum>Qt::NoFocus</enum>
	  </property>
         <property name="toolTip">
          <string>{</string>
         </property>
         <property name="text">
          <string>CH-</string>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="mute">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="text">
        <string>Muting</string>
       </property>
       <property name="toolTip">
        <string>U</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="recall">
	<property name="focusPolicy">
	  <enum>Qt::NoFocus</enum>
	</property>
       <property name="toolTip">
        <string>A</string>
       </property>
       <property name="text">
        <string>Recall</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
